packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx

```diff
@@ -1,19 +1,87 @@
-import React from 'react';
+import {
+  EmptyStateComingSoon,
+  EncounterList,
+  EncounterListColumn,
+  PatientChartProps,
+  SummaryCard,
+  SummaryCardColumn,
+  getObsFromEncounter,
+} from '@ohri/openmrs-esm-ohri-commons-lib';
+import { useConfig } from '@openmrs/esm-framework';
+import React, { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { EmptyStateComingSoon } from '@ohri/openmrs-esm-ohri-commons-lib';
+import { moduleName } from '../../..';
 
-interface TptPatientSummaryProps {
-  patientUuid: string;
-}
-
-const TptPatientSummary: React.FC<TptPatientSummaryProps> = ({ patientUuid }) => {
+const TptPatientSummary: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
+  const config = useConfig();
+
+  const headerRecentTPT = t('recentTpt', 'Recent TPT Cases');
+  const headerPreviousCases = t('previousCases', 'Previous TPT Cases');
```

Comment:
change the key names to include the 3 words `recentTptCases` and `previousTptCases`